rules/config

```diff
@@ -125,8 +125,8 @@ SONIC_DPKG_CACHE_SOURCE ?= /var/cache/sonic/artifacts
 DEFAULT_VS_PREPARE_MEM = yes
 
 
-# INCLUDE_SYSTEM_TELEMETRY - build docker-sonic-telemetry for system telemetry support
-INCLUDE_SYSTEM_TELEMETRY = y
+# INCLUDE_SYSTEM_GNMI - build docker-sonic-gnmi for system gnmi support
+INCLUDE_SYSTEM_GNMI = y
```

Comment:
>INCLUDE_SYSTEM_GNMI [](http://example.com/codeflow?start=0&length=19)

Will you add another `INCLUDE_SYSTEM_TELEMETRY` flag and default to `n`?

Reply:
# INCLUDE_SYSTEM_GNMI - build docker-sonic-gnmi for system gnmi support

This flag is used to build docker image, now we don't have docker-sonic-telemetry, do you think we still need INCLUDE_SYSTEM_TELEMETRY?